src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/HttpRequestManager.cs

```diff
@@ -65,6 +65,11 @@ public async Task<HttpResponseMessage> ExecuteAsync(int retryCount = 3)
 
                             retriesRemaining = 0;
                         }
+                        else if (response.StatusCode == HttpStatusCode.BadRequest)
```

Comment:
Is the plan to retry if this happens? Are we calling with 0 retries? If not, should we set `retriesRemaining` to 0?

Reply:
I don't think so. That error means something is wrong in the request body. IMO retrying won't do anything.